core/mo-services-impl/ccsds-com-impl/src/main/java/esa/mo/com/impl/consumer/ArchiveSyncGenAdapter.java

```diff
@@ -41,9 +41,9 @@
     private final Map<Integer, byte[]> receivedChunks;
     private final Semaphore completed;
 //    private Long interactionTicket = null;
-    private UInteger numberOfChunks = null;
-    private long lastTimeReceived = 0;
-    private long lastknowIndex = 0;
+    private UInteger numberOfChunks;
+    private long lastTimeReceived;
+    private long lastknowIndex;
```

Comment:
These changes make the code less readable. The zeros make it explicit that the initialization IS zero.